suprb/optimizer/rule/es/es.py

```diff
@@ -122,4 +122,6 @@ def _optimize(self, X: np.ndarray, y: np.ndarray, initial_rule: Rule, random_sta
                     elitist = elitists[0]
                     break
 
+            mutation.adapt(elitist.fitness_)
```

Comment:
we could check if mutation has a member adapt and if so call it to remove the need to have an empty adapt on most of our mutation operators

Reply:
done